[MeshingApplication] Compilation issue

**Description**
Dear all,

I just tried to compile the `MeshingApp` in windows (in linux is the same) and I got the following compilation errors:

~~~
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(643): error C3861: 'MMG2D_G
et_quadrilateral': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication\
KratosMeshingCore.vcxproj]
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(1408): error C3861: 'MMG2D_
Get_quadrilateral': no se encontró el identificador [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication
\KratosMeshingCore.vcxproj]
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(1695): error C2065: 'MMGS_I
PARAM_optim': identificador no declarado [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication\KratosMesh
ingCore.vcxproj]
D:\KRATOS_MULTIPHYSICS\Kratos\applications\MeshingApplication\custom_utilities\mmg\mmg_utilities.cpp(1709): error C2660: 'MMG2D_
Set_meshSize': la función no acepta 5 argumentos [D:\KRATOS_MULTIPHYSICS\Kratos\build\Release\applications\MeshingApplication\Kr
atosMeshingCore.vcxproj]
~~~

among many more...

Should I update anything?

Regards,

@JoseMGonzalez-UPC 

I attach my configure.bat:

~~~ bat
rem Please do not modify this script

rem For any question please contact with us in:
rem  - https://github.com/KratosMultiphysics/Kratos

rem Optional parameters:
rem You can find a list will all the compiation options in INSTALL.md or here:
rem  - https://github.com/KratosMultiphysics/Kratos/wiki/Compilation-options

call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat"

rem Set compiler
@echo off
set CC=cl.exe
set CXX=cl.exe

rem Set variables
set KRATOS_SOURCE=~0,-1%/..
set KRATOS_BUILD=%KRATOS_SOURCE%/build
set KRATOS_APP_DIR=applications
set KRATOS_INSTALL_PYTHON_USING_LINKS=ON

rem Set basic configuration
set KRATOS_BUILD_TYPE=Release
set BOOST_ROOT=D:\boost_1_69_0
set PYTHON_EXECUTABLE=C:\Program Files\Python310\python.exe

rem Set applications to compile
set KRATOS_APPLICATIONS=
CALL :add_app %KRATOS_APP_DIR%\LinearSolversApplication;
CALL :add_app %KRATOS_APP_DIR%\StructuralMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\ConstitutiveLawsApplication;
rem CALL :add_app %KRATOS_APP_DIR%\ContactStructuralMechanicsApplication;
CALL :add_app %KRATOS_APP_DIR%\FemToDemApplication;
CALL :add_app %KRATOS_APP_DIR%\PfemFluidDynamicsApplication;
CALL :add_app %KRATOS_APP_DIR%\DelaunayMeshingApplication;
CALL :add_app %KRATOS_APP_DIR%\DEMApplication;
CALL :add_app %KRATOS_APP_DIR%\MeshingApplication;
CALL :add_app %KRATOS_APP_DIR%\DemStructuresCouplingApplication;

rem Clean
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\cmake_install.cmake"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeCache.txt"
del /F /Q "%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%\CMakeFiles"

rem Configure
@echo on
cmake -G"Visual Studio 15 2017 Win64" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"  ^
-DINCLUDE_FEAST=OFF                                                      ^
-DINCLUDE_MMG=ON                                                         ^
-DUSE_TRIANGLE_NONFREE_TPL=ON                                            ^
-DMMG_BUILDDIR="D:/mmg_master/buid"                                      ^
-DMMG_LIBDIR="D:/mmg_master/buid/lib/Release"                            ^
-DCMAKE_CXX_FLAGS=" -D_SCL_SECURE_NO_WARNINGS /MP5"                      ^
-DUSE_EIGEN_MKL=ON                                                       ^
-DINSTALL_RUNKRATOS=OFF                                                  ^
-DUSE_COTIRE=OFF                                                         ^
-DKRATOS_BUILD_TESTING=ON

rem Build
cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target install -- /property:configuration=%KRATOS_BUILD_TYPE% /p:Platform=x64
goto:eof

rem Function to add apps
:add_app
set KRATOS_APPLICATIONS=%KRATOS_APPLICATIONS%%1;
goto:eof

~~~

Hi, 
I've the same problem here under ubuntu.
When trying to install the MeshingApplication with mmg.
Regarding mmg, I installed the latest version 5.7 using the script:
```
#!/bin/bash
# Compiling mmg
export CC=${CC:-gcc}
export CXX=${CXX:-g++}

# Configure
cmake -D CMAKE_INSTALL_PREFIX=/path2mmg                   \
        -D CMAKE_BUILD_TYPE=Release                                   \
        -D CMAKE_C_COMPILER=${CC}                                     \
        -D CMAKE_CXX_COMPILER=${CXX}                                  \
        -D CMAKE_C_FLAGS="-O3 -march=native -mtune=native -fopenmp"   \
        -D CMAKE_CXX_FLAGS="-O3 -march=native -mtune=native -fopenmp" \
        -D USE_SCOTCH=OFF                                             \
        -D LIBMMG_SHARED=ON                                           \
        -D LIBMMG_STATIC=OFF                                          \
        -D LIBMMGS_SHARED=ON                                          \
        -D LIBMMGS_STATIC=OFF                                         \
        -D LIBMMG2D_SHARED=ON                                         \
        -D LIBMMG2D_STATIC=OFF                                        \
        -D LIBMMG3D_SHARED=ON                                         \
        -D LIBMMG3D_STATIC=OFF                                        \
        -D BUILD_SHARED_LIBS=ON                                       \

# Build
cmake --build $(pwd) -- -j$(nproc)
```
Compilation is ok and files are written in 
path2mmg/include
path2mmg/lib
path2mmg/bin

Then I modified the standard_configuration.sh to install kratos:

```
# Function to add apps
add_app () {
    export KRATOS_APPLICATIONS="${KRATOS_APPLICATIONS}$1;"
}

# Set compiler
export CC=${CC:-gcc}
export CXX=${CXX:-g++}

# Set variables
export KRATOS_SOURCE="/mypath/Kratos"
export KRATOS_BUILD="${KRATOS_SOURCE}/build"
export KRATOS_APP_DIR="${KRATOS_SOURCE}/applications"
export MMG_ROOT="path2mmg"

# export KRATOS_INSTALL_PYTHON_USING_LINKS=ON

# Set basic configuration
export KRATOS_BUILD_TYPE=${KRATOS_BUILD_TYPE:-"Release"}
export PYTHON_EXECUTABLE=${PYTHON_EXECUTABLE:-"/usr/bin/python3"}

# Set applications to compile
export KRATOS_APPLICATIONS=
add_app ${KRATOS_APP_DIR}/LinearSolversApplication
add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication
add_app ${KRATOS_APP_DIR}/FluidDynamicsApplication
add_app ${KRATOS_APP_DIR}/MeshingApplication

# Clean
clear
rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/cmake_install.cmake"
rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeCache.txt"
rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"

# Configure
cmake -H"${KRATOS_SOURCE}" \
        -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" \
        -D USE_MPI=OFF                                 \
        -D USE_EIGEN_MKL=OFF                           \
        -D KRATOS_GENERATE_PYTHON_STUBS=ON             \
        -D INCLUDE_MMG=ON                              \
        -D MMG_ROOT=${MMG_ROOT}                        \
        -D MMG_INCLUDE_DIRS="${MMG_ROOT}/include"                \
        -D MMG2D_INCLUDE_DIRS="${MMG_ROOT}/include/mmg/mmg2d/"  \
        -D MMG3D_INCLUDE_DIRS="${MMG_ROOT}/include/mmg/mmg3d/"  \
        -D MMGS_INCLUDE_DIRS="${MMG_ROOT}/include/mmg/mmgs/"    \
        -D MMG_LIBRARIES="${MMG_ROOT}/lib/libmmg.so"              \
        -D MMG2D_LIBRARIES="${MMG_ROOT}/lib/libmmg2d.so"          \
        -D MMG3D_LIBRARIES="${MMG_ROOT}/lib/libmmg3d.so"          \
        -D MMGS_LIBRARIES="${MMG_ROOT}/lib/libmmgs.so"            \

# Build
cmake --build "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" --target install -- -j$(nproc)
```
It fails to install and looks like it does not find some headers (he does find the .so files) as you can read in the following :
```
-- The C compiler identification is GNU 11.3.0
-- The CXX compiler identification is GNU 11.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Deprecation Warning at CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- No KRATOS_SOURCE_DIR is defined, using: /home/clouzetpierre/Documents/src/Kratos
-- No KRATOS_BINARY_DIR is defined, using: /home/clouzetpierre/Documents/src/Kratos/build/Release
-- No INSTALL_RUNKRATOS is defined, setting to ON
-- No KRATOS_INSTALL_PYTHON_USING_LINKS is defined, setting to OFF
-- No CMAKE_INSTALL_MESSAGE is defined. Setting to NEVER
-- Found Git: /usr/bin/git (found version "2.34.1") 
-- Additional default options were set for GCC
-- CMAKE_SYSTEM_NAME = Linux
-- CMAKE_CXX_COMPILER_ID = GNU
-- CMAKE_CXX_FLAGS =  -funroll-loops -Wall -Wl,--no-as-needed -ldl -Wsuggest-override
-- CMAKE_C_FLAGS =  -funroll-loops -Wall -Wl,--no-as-needed -ldl
-- Found PythonInterp: /usr/bin/python3 (found version "3.10.6") 
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.10.so
-- LTO is Disabled
AR VERSION: /usr/bin/ar
-- Python version used for the interface will be 3.10
-- Standard install dir /home/clouzetpierre/Documents/src/Kratos/bin/Release
"KRATOS_SHARED_MEMORY_PARALLELIZATION" not defined, defaulting to "OpenMP"
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
Using OpenMP for shared memory parallelization
-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.74.0/BoostConfig.cmake (found version "1.74.0")  
-- Boost Include: /usr/include
-- Boost Linkdir: 
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.11") 
-- AMatrix not found. Please define the AMATRIX_DIR pointing to the AMatrix root directory.
 
CMAKE_BUILD_TYPE ........ Release
 
WARNING: YOUR GIDPOST IS BEING COMPILED  WITHOUT DEBUG (ALL ASSERTS WILL BE AVOIDED)
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Configuring applications (ENV):
Adding application '/home/clouzetpierre/Documents/src/Kratos/applications/LinearSolversApplication'
**** configuring KratosLinearSolversApplication ****
Adding application '/home/clouzetpierre/Documents/src/Kratos/applications/StructuralMechanicsApplication'
**** configuring KratosStructuralMechanicsApplication ****
Adding application '/home/clouzetpierre/Documents/src/Kratos/applications/FluidDynamicsApplication'
**** configuring KratosFluidDynamicsApplication ****
Adding application '/home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication'
**** configuring KratosMeshingApplication ****
-- Searching for MMG in MMG_BUILDDIR: /home/clouzetpierre/Documents/src/mmg
-- ... and MMG_LIBDIR: /home/clouzetpierre/Documents/src/mmg/lib
-- A cache variable, namely MMG_DIR, has been set to specify the install directory of MMG
-- Looking for mmg -- libmmgtypes.h not found
-- Looking for MMG5_loadMshMesh_part1
-- Looking for MMG5_loadMshMesh_part1 - found
-- Found MMG: /home/clouzetpierre/Documents/src/mmg/lib/libmmg.so  
-- A cache variable, namely MMG_DIR, has been set to specify the install directory of MMG
-- Looking for mmg2d -- libmmgtypes.h not found
-- Looking for MMG5_loadMshMesh_part1
-- Looking for MMG5_loadMshMesh_part1 - found
-- Found MMG2D: /home/clouzetpierre/Documents/src/mmg/lib/libmmg2d.so  
-- A cache variable, namely MMG_DIR, has been set to specify the install directory of MMG
-- Looking for mmg3d -- libmmgtypes.h not found
-- Looking for MMG5_loadMshMesh_part1
-- Looking for MMG5_loadMshMesh_part1 - found
-- Found MMG3D: /home/clouzetpierre/Documents/src/mmg/lib/libmmg3d.so  
-- A cache variable, namely MMG_DIR, has been set to specify the install directory of MMG
-- Looking for mmgs -- libmmgtypes.h not found
-- Looking for MMG5_loadMshMesh_part1
-- Looking for MMG5_loadMshMesh_part1 - found
-- Found MMGS: /home/clouzetpierre/Documents/src/mmg/lib/libmmgs.so  
-- Compilation with MMG: /home/clouzetpierre/Documents/src/mmg/lib/libmmg.so MMG_INCLUDE_DIRS-NOTFOUND
-- Compilation with MMG2D: /home/clouzetpierre/Documents/src/mmg/lib/libmmg2d.so MMG2D_INCLUDE_DIRS-NOTFOUND
-- Compilation with MMG3D: /home/clouzetpierre/Documents/src/mmg/lib/libmmg3d.so MMG3D_INCLUDE_DIRS-NOTFOUND
-- Compilation with MMGS: /home/clouzetpierre/Documents/src/mmg/lib/libmmgs.so MMGS_INCLUDE_DIRS-NOTFOUND

***********************************************************************

        LIST OF APPLICATIONS THAT ARE GOING TO BE COMPILED:

        LinearSolversApplication
        StructuralMechanicsApplication
        FluidDynamicsApplication
        MeshingApplication

        APPLICATIONS ADDED TO COMPILATION THROUGH DEPENDENCIES:


***********************************************************************

List of upcoming API-breaking and behavior changes:
	 Geometry 'Dimension' deprecation: https://github.com/KratosMultiphysics/Kratos/pull/10445
	 Geometry old projection methods deprecation: https://github.com/KratosMultiphysics/Kratos/pull/9024
	 Geometry old closest point methods deprecation: https://github.com/KratosMultiphysics/Kratos/pull/9243
	 Model 'CreateModelPart' behavior change (no error thrown if model part exists): https://github.com/KratosMultiphysics/Kratos/pull/9598
	 Make GetIntegrationMethod method of Condition const: https://github.com/KratosMultiphysics/Kratos/pull/9769
	 Behavior change of ModelPart.GetProperties (MeshIndex removed): https://github.com/KratosMultiphysics/Kratos/pull/9774
-- clipper : /home/clouzetpierre/Documents/src/Kratos/external_libraries/clipper
-- tinyexpr : /home/clouzetpierre/Documents/src/Kratos/external_libraries/tinyexpr/tinyexpr/
python libraries to be installed are: /usr/lib/x86_64-linux-gnu/libpython3.10.so.1.0
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
MMG2D_INCLUDE_DIRS
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
MMG3D_INCLUDE_DIRS
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
MMGS_INCLUDE_DIRS
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
MMG_INCLUDE_DIRS
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication
   used as include directory in directory /home/clouzetpierre/Documents/src/Kratos/applications/MeshingApplication

CMake Warning (dev) in applications/MeshingApplication/CMakeLists.txt:
  Policy CMP0021 is not set: Fatal error on relative paths in
  INCLUDE_DIRECTORIES target property.  Run "cmake --help-policy CMP0021" for
  policy details.  Use the cmake_policy command to set the policy and
  suppress this warning.

  Found relative path while evaluating include directories of
  "KratosMeshingCore":

    "MMG_INCLUDE_DIRS-NOTFOUND"

This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) in applications/MeshingApplication/CMakeLists.txt:
  Policy CMP0021 is not set: Fatal error on relative paths in
  INCLUDE_DIRECTORIES target property.  Run "cmake --help-policy CMP0021" for
  policy details.  Use the cmake_policy command to set the policy and
  suppress this warning.

  Found relative path while evaluating include directories of
  "KratosMeshingCore":

    "MMG2D_INCLUDE_DIRS-NOTFOUND"

This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) in applications/MeshingApplication/CMakeLists.txt:
  Policy CMP0021 is not set: Fatal error on relative paths in
  INCLUDE_DIRECTORIES target property.  Run "cmake --help-policy CMP0021" for
  policy details.  Use the cmake_policy command to set the policy and
  suppress this warning.

  Found relative path while evaluating include directories of
  "KratosMeshingCore":

    "MMG3D_INCLUDE_DIRS-NOTFOUND"

This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) in applications/MeshingApplication/CMakeLists.txt:
  Policy CMP0021 is not set: Fatal error on relative paths in
  INCLUDE_DIRECTORIES target property.  Run "cmake --help-policy CMP0021" for
  policy details.  Use the cmake_policy command to set the policy and
  suppress this warning.

  Found relative path while evaluating include directories of
  "KratosMeshingCore":

    "MMGS_INCLUDE_DIRS-NOTFOUND"

This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error in applications/MeshingApplication/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "KratosMeshingApplication":

    "MMG_INCLUDE_DIRS-NOTFOUND"



-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
Consolidate compiler generated dependencies of target KratosVersion
Consolidate compiler generated dependencies of target tinyexpr
Consolidate compiler generated dependencies of target gidpost
Consolidate compiler generated dependencies of target runkratos
Consolidate compiler generated dependencies of target clipper
Consolidate compiler generated dependencies of target tetMeshOpt
etc etc unfil fatal error
```

It looks like its not taking into account some path like the MMG_INCLUDE_DIRS. I tried to set them manually based on what vraiables I found in the different CMakeLists.txt

--- a/cmake_modules/FindMMG2D.cmake
+++ b/cmake_modules/FindMMG2D.cmake
@@ -90,9 +90,9 @@ if(MMG_INCDIR)
   find_path(MMG2D_libmmgtypes.h_DIRS
     NAMES libmmgtypes.h
     HINTS ${MMG_INCDIR}
-    PATH_SUFFIXES "mmg/mmg2d" "mmg2d")
+    PATH_SUFFIXES "mmg" "mmg/mmg2d" "mmg2d" "mmg/common")
 else()
-  if(MMG_DIR)
+    if(MMG_DIR)
     set(MMG2D_libmmgtypes.h_DIRS "MMG2D_libmmgtypes.h_DIRS-NOTFOUND")
     find_path(MMG2D_libmmgtypes.h_DIRS
       NAMES libmmgtypes.h

--- a/cmake_modules/FindMMG2D.cmake
+++ b/cmake_modules/FindMMG2D.cmake
@@ -92,7 +92,7 @@ if(MMG_INCDIR)
     HINTS ${MMG_INCDIR}
     PATH_SUFFIXES "mmg" "mmg/mmg2d" "mmg2d" "mmg/common")
 else()
-    if(MMG_DIR)
+  if(MMG_DIR)
     set(MMG2D_libmmgtypes.h_DIRS "MMG2D_libmmgtypes.h_DIRS-NOTFOUND")
     find_path(MMG2D_libmmgtypes.h_DIRS
       NAMES libmmgtypes.h

--- a/cmake_modules/FindMMG3D.cmake
+++ b/cmake_modules/FindMMG3D.cmake
@@ -90,7 +90,7 @@ if(MMG_INCDIR)
   find_path(MMG3D_libmmgtypes.h_DIRS
     NAMES libmmgtypes.h
     HINTS ${MMG_INCDIR}
-    PATH_SUFFIXES "mmg/mmg3d" "mmg3d")
+    PATH_SUFFIXES "mmg/mmg3d" "mmg3d" "mmg/common")
 else()
   if(MMG_DIR)
     set(MMG3D_libmmgtypes.h_DIRS "MMG3D_libmmgtypes.h_DIRS-NOTFOUND")

--- a/cmake_modules/FindMMGS.cmake
+++ b/cmake_modules/FindMMGS.cmake
@@ -90,7 +90,7 @@ if(MMG_INCDIR)
   find_path(MMGS_libmmgtypes.h_DIRS
     NAMES libmmgtypes.h
     HINTS ${MMG_INCDIR}
-    PATH_SUFFIXES "mmg/mmgs" "mmgs")
+    PATH_SUFFIXES "mmg/mmgs" "mmgs" "mmg/common")
 else()
   if(MMG_DIR)
     set(MMGS_libmmgtypes.h_DIRS "MMGS_libmmgtypes.h_DIRS-NOTFOUND")

